main.go

```diff
@@ -201,16 +198,13 @@ func main() {
 		glog.Infof("Using %s namespace", options.namespace)
 	}
 
-	if isNotExists(options.kubeconfig) && !(options.inCluster) {
-		glog.Fatalf("kubeconfig invalid and --in-cluster is false; kubeconfig must be set to a valid file(kubeconfig default file name: $HOME/.kube/config)")
-	}
-	if options.apiserver != "" {
-		glog.Infof("apiserver set to: %v", options.apiserver)
+	if options.apiserver == "" && options.kubeconfig == "" {
```

Comment:
These two lines are unnecessary as [`BuildConfigFromFlags`](https://github.com/kubernetes/client-go/blob/master/tools/clientcmd/client_config.go#L531) will check and log this.